src/univariate/continuous/rayleigh.jl

```diff
@@ -89,3 +89,17 @@ quantile(d::Rayleigh, p::Real) = sqrt(-2d.σ^2 * log1p(-p))
 #### Sampling
 
 rand(rng::AbstractRNG, d::Rayleigh) = d.σ * sqrt(2 * randexp(rng))
+
+
+#### Fitting
+
+function fit_mle(::Type{<:Rayleigh}, x::AbstractArray{T}) where T<:Real
```

Comment:
```suggestion
function fit_mle(::Type{<:Rayleigh}, x::AbstractArray{T}) where {T<:Real}
```
for consistent style